crates/moon/src/cli/coverage.rs

```diff
@@ -61,8 +62,13 @@ pub struct CoverageSubcommand {
 
 #[derive(Debug, clap::Parser)]
 pub struct CoverageAnalyzeSubcommand {
-    #[clap(flatten)]
-    pub test_flags: Box<TestSubcommand>,
+    /// Analyze coverage for a specific package.
+    #[clap(short, long)]
+    package: Option<String>,
+
+    /// Extra flags passed directly to `moon test`
+    #[clap(short, long, hide = true, allow_hyphen_values = true)]
```

Comment:
I personally don't think `hide` is really a good idea, as this command doesn't advertise a direct correspondence with the more "plumbing" commands, and hiding this will require anyone who want to slightly alter the behavior (e.g. use a different backend) fall hard on the raw command combination.

Reply:
We do need some form of precise control in the future. However hiding the flag here means we have no guarantee of the shape of this API.